src/inputs/EditableLabel/EditableLabel.js

```diff
@@ -108,33 +112,38 @@ export const EditableLabel = (props) => {
   return (
     <div data-cy="editable-label">
       {isEditing ? (
-        <TextField
-          className={classes.textField}
-          autoFocus
-          defaultValue={value}
-          error={error != null}
-          helperText={error}
-          placeholder={labels.title}
-          onChange={checkNewValueOnChange}
-          onBlur={(event) => {
-            stopEdition(event);
-          }}
-          onClick={(event) => event.stopPropagation()}
-          onKeyPress={(event) => {
-            if (event.key === 'Enter') {
+        <FormControl>
+          <OutlinedInput
+            id="component-outlined"
+            className={classes.OutlinedInput}
+            autoFocus
+            defaultValue={value}
+            error={error != null}
+            placeholder={labels.title}
+            onChange={checkNewValueOnChange}
+            onBlur={(event) => {
               stopEdition(event);
-            }
-          }}
-          onKeyDown={(event) => {
-            if (event.key === 'Escape') {
-              setEditing(false);
-            }
-          }}
-          onFocus={(event) => {
-            event.stopPropagation();
-          }}
-          {...textFieldProps}
-        />
+            }}
+            onClick={(event) => event.stopPropagation()}
+            onKeyPress={(event) => {
+              if (event.key === 'Enter') {
+                stopEdition(event);
+              }
+            }}
+            onKeyDown={(event) => {
+              if (event.key === 'Escape') {
+                setEditing(false);
+              }
+            }}
+            onFocus={(event) => {
+              event.stopPropagation();
+            }}
+            {...textFieldProps}
```

Comment:
The name `textFieldProps` is no longer relevant if the TextField has been removed (and replaced here by an `OutlinedInput`. Do we still need additional props for this element ?